src/internal/storage/fileset/unordered_writer.go

```diff
@@ -20,7 +20,6 @@ type UnorderedWriter struct {
 	memAvailable, memThreshold int64
 	fileThreshold              int64
 	buffer                     *Buffer
-	subFileSet                 int64
```

Comment:
I'm curious to know what this was used for before

Reply:
This is a vestige from when we needed a unique and ordered value for each primitive file set. That was how we knew the order of the file set layers. The current model, that has been in place for a while, uses lists both in-memory and in the serialization format.